packages/core/src/utils/shell-utils.ts

```diff
@@ -301,7 +301,9 @@ export function checkCommandPermissions(
       return {
         allAllowed: false,
         disallowedCommands,
-        blockReason: `Command(s) not on the global or session allowlist.`,
+        blockReason: `Command(s) not on the global or session allowlist. Disallowed commands: ${disallowedCommands
```

Comment:
Nit: Command(s) not on the global or session allowlist. Disallowed commands
```suggestion
        blockReason: `Command(s) not on the global or session allowlist or exist in disallowed commands. Disallowed commands: ${disallowedCommands
```